couchdbkit/schema/base.py

```diff
@@ -130,8 +131,13 @@ def save(self, **params):
         self.validate()
         db = self.get_db()
 
+        for prop_name, prop in self.properties().iteritems():
+            if isinstance(prop, p.DateTimeProperty) and prop.auto_now_add:
+                setattr(self, prop_name, datetime.utcnow())
```

Comment:
if we're going to rely on this for pulling source data for the warehouse i think we need to support it for bulk save too